packages/components/src/components/expansion-panel/expansion-panel.jsx

```diff
@@ -0,0 +1,102 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import Icon from '../icon';
+
+class ExpansionPanel extends React.Component {
+    state = {
+        open_id: [],
+        isOpen: false,
+    };
+
+    onClick = () => {
+        // close if clicking the expansion panel that's open, otherwise open the new one
+        this.setState({ isOpen: !this.state.isOpen });
+    };
+
+    onArrayItemClick = id => {
+        this.setState(state => {
+            if (state.open_id.includes(id)) {
+                const open_id = state.open_id.filter(item => {
+                    return item !== id;
+                });
```

Comment:
- `item` could be renamed to `current_id` since we're dealing with ids, not items.
- You could move the return value right after the arrow function, e.g.:

```suggestion
                const open_id = state.open_id.filter(item => item !== id);
```